scripts/ci/pre_commit_check_license.sh

```diff
@@ -15,9 +15,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-set -uo pipefail
```

Comment:
No ` set -euo pipefail ` here?

Reply:
Updated it to source the ci_scripts instead of running them - this way set -euo and exit on error will come from the _utils.sh